opencti-platform/opencti-graphql/src/modules/notification/notification.graphql

```diff
@@ -171,6 +164,7 @@ input TriggerLiveAddInput {
     event_types: [TriggerEventType!]!
     outcomes: [StixRef!]!
     filters: String
+    recipients: [String!]
```

Comment:
Why an array when we have checks that we want an array with one entry?

Reply:
it's for the next part of the issue : we will want to create triggers for multiples users